cli/src/commands/create/source.rs

```diff
@@ -62,6 +67,7 @@ pub fn create(client: &Client, args: &CreateSourceArgs, printer: &Printer) -> Re
                 should_translate: *should_translate,
                 bucket_id,
                 sensitive_properties: None,
+                source_type: source_type.as_deref(),
```

Comment:
With the changes above, you'd need to call `.as_ref()` on `source_type` here to go from 
`Option<SourceType>` to `Option<& SourceType>`
```suggestion
                source_type: source_type.as_ref(),
```

Reply:
Right :eyes:, I don't really get what's going on here :see_no_evil: